Libs/MRML/Core/vtkMRMLTransformableNode.cxx

```diff
@@ -141,6 +141,45 @@ void vtkMRMLTransformableNode::ProcessMRMLEvents ( vtkObject *caller,
     }
 }
 
+//---------------------------------------------------------------------------
+void vtkMRMLTransformableNode
+::OnNodeReferenceAdded(vtkMRMLNodeReference* reference)
+{
+  Superclass::OnNodeReferenceAdded(reference);
+  if (std::string(reference->GetReferenceRole()) == this->TransformNodeReferenceRole)
+    {
+    this->InvokeCustomModifiedEvent(
+      vtkMRMLTransformableNode::TransformModifiedEvent, reference->GetReferencedNode());
+    }
+
+  vtkMRMLTransformNode* transformNode =
+    vtkMRMLTransformNode::SafeDownCast(reference->GetReferencedNode());
+  if (transformNode)
+    {
+    transformNode->InvokeEvent(
+      vtkMRMLTransformNode::TransformReferenceModifiedEvent, this);
```

Comment:
Yes, doing it in a separate pull request could make sense. Till then, remove the automatic observation of transform chain updates and determine the boundary when the editing widget is turned on (and maybe when you enter the Transforms module and when a transforms is applied in the transforms module). 

We could even keep this behavior in the long term: update bounds when you show the widget (and maybe add a button that would update the bounds so that the user does not have to hide/show the widget to update bounds).